apps/dotcms-ui/src/app/portlets/dot-templates/dot-template-list/dot-template-list.component.spec.ts

```diff
@@ -344,6 +344,16 @@ describe('DotTemplateListComponent', () => {
         dotSiteBrowserService = TestBed.inject(DotSiteBrowserService);
     });
 
+    it('should reload portlet only when the site change', () => {
+        let checkbox: Checkbox = fixture.debugElement.query(
```

Comment:
use `data-testid` for testing

Reply:
done https://github.com/dotCMS/core-web/pull/1931/commits/be0eb22fbc68fd61458f83bf116730d021224821#diff-8010dccdb9bffcf525460e8f6f73538ef76724fe6f9a8baddd199bfed71fc1e3R20